tests/unit/test_column_noise.py

```diff
@@ -311,11 +312,11 @@ def test_leave_blank(dataset: Dataset, fuzzy_checker: FuzzyChecker) -> None:
                 },
             },
         }
-    )[DATASET_SCHEMAS.census.name][Keys.COLUMN_NOISE]["zipcode"][NOISE_TYPES.leave_blank.name]
+    )
 
-    data = dataset.data[["numbers"]]
```

Comment:
What is the purpose of changing the test from working on the dummy "numbers" column to "zipcode"?

Reply:
The test would previously work with a column name in the config that didn't appear in the config but not anymore. Since we pass in the whole config instead of at some arbitrary depth now when noising, we need all the information in the config to be consistent with the dataset.